css/properties/appearance.json

```diff
@@ -258,26 +217,32 @@
                 "version_added": false
               },
               "opera": {
-                "version_added": "15"
+                "version_added": "15",
+                "partial_implementation": true
```

Comment:
Can you say a bit about why this is considered partial? Is just because button was folded in? Notes would be helpful here, so one can tell when it should set to fully supported again, and so readers need not guess about what will work.

Reply:
Opera 15 corresponds to an early version of Chromium which doesn't follow the spec and doesn't support `auto` etc. However, whichever version of Opera that maps to Chromium 84 should have full support. https://help.opera.com/en/opera-version-history/ only shows up to Chromium 81.

Reply:
Fixed in 66b2ab0

I'm not sure how to explain the partial implementation in a short note. "Values can be applied to any element, rather than only widgets as per spec."?

Reply:
I usually check `browsers/opera.json` to find the matching release. But when preparing changing, you can also run something like `npm run mirror -- --modify=always opera css` and see what changes it makes.

Reply:
👍 on adding notes for `partial_implementation`. That text is probably OK, but if you can point to an example or longer-form description of the problem, I can probably provide more detailed help on writing a note (it's definitely a challenging format for tech writing).